storagegroup/grpc/types.go

```diff
@@ -15,6 +15,8 @@ func (m *StorageGroup) SetValidationHash(v *refs.Checksum) {
 }
 
 // SetExpirationEpoch sets number of the last epoch of the storage group lifetime.
+//
+// Deprecated: do not use, `expiration_epoch` field is deprecated in protocol.
```

Comment:
I'm at a loss as to whether there should be such a comment: any code must set it anyway since we use exactly that field everywhere in SN, IR apps. SDK also [sets/requires](https://github.com/nspcc-dev/neofs-sdk-go/blob/f2f97f656da7f72216bc7ed527aabe50a4bc6fc7/storagegroup/storagegroup.go#L276-L282) that field.

Reply:
From one side, I don't see any protocol statements which make `ReadFromObject` to require `Object must contain unambiguous information about its expiration epoch`.

From the other side, the method is deprecated in order to prevent "new" clients to use it. This does not cancel calling it on node side for backward compatibility.